Localize/lang/strings.json

```
@@ -683,6 +685,7 @@
   "SgiTAh": "Please enter your input",
```

Don't use "please" unless customer is asked to take an inconvenient action because the system failed:

```suggestion
  "SgiTAh": "Enter your input.",
```